src/LineageTree/lineageTree.py

```diff
@@ -2662,13 +2666,13 @@ def calculate_dtw(
         if fast:
             if centered_band:
                 slope, intercept = self.__calculate_diag_line(dist_mat)
-                path, cost_mat = self.__dp(dist_mat, start_d, back_d, w, slope, intercept, use_absolute=False)
+                path, cost_mat, final_cost = self.__dp(dist_mat, start_d, back_d, w, slope, intercept, use_absolute=False)
             else:
-                path, cost_mat = self.__dp(dist_mat, start_d, back_d, w, use_absolute=True)
+                path, cost_mat, final_cost = self.__dp(dist_mat, start_d, back_d, w, use_absolute=True)
```

Comment:
Is it normal that the `use_absolute` variable is hard coded?

Reply:
what do you mean, hard coded?